lib/rules/string-no-newline/README.md

```diff
@@ -70,3 +70,33 @@ a {
   font-family: "Times New Roman";
 }
 ```
+
+## Optional secondary options
+
+### `ignore: ["at-rules", "declarations"]`
+
+#### `"at-rules"`
+
+Ignore strings in at-rules.
+
+The following patterns are _not_ considered problems:
+
+<!-- prettier-ignore -->
+```css
+@import url('example
+.com');
+```
+
+#### `"declarations"`
+
+Ignore strings in declarations.
+
+The following patterns are _not_ considered problems:
+
+<!-- prettier-ignore -->
+```css
+a {
+  content: "first
+    second";
```

Comment:
```suggestion
  content: "foo
    bar";
```

Reply:
Should I change the other ones too?
See: [string-no-newline | Stylelint](https://stylelint.io/user-guide/rules/string-no-newline/)

Reply:
We can do that in a follow-up PR if we've time. Many examples and tests predate [our conventions](https://stylelint.io/developer-guide/rules#write-tests). Ideally, we'd tidy them all up in some refactor and doc PRs, but it's mundane work. However, we can ensure that _new_ tests and examples follow the conventions.

Reply:
No need to revert.